src/containers/booklist.js

```diff
@@ -2,8 +2,9 @@ import React from 'react';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import Book from '../components/Book';
+import { removeBook } from '../actions';
 
-const BookList = ({ books }) => (
+const BookList = ({ books, removeBook }) => (
```

Comment:
* [ ] You should implement `handleBookRemove` here [screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone3/ImplementsRemoveBook.png)
```suggestion
const BookList = ({ books, removeBook }) => {
   const handleBookRemove = (book) => removeBook(book)
```